Code/Core/Containers/Array.h

```diff
@@ -12,6 +12,8 @@
 #include "Core/Math/Conversions.h"
 #include "Core/Mem/Mem.h"
 
+#include <initializer_list>
```

Comment:
FASTBuild doesn't use the C++ standard library at all, so this header can not be included.

I see it's use to initialize a couple of arrays which are then passed to functions.

I can think of two possible ways to resolve this:
1) Implement a custom InitializeList equivalent (just as FASTBuild implements its own Move to replace std::move)
2) Switch to a c-style array for the uses in this PR

Note that while the initializer_list syntax is convenient, it's very inefficient (though that doesn't matter particularly for this use-case)

Reply:
I have implemented a util function with varadic template, not a complete replacement of initializer_list in 98ef95275848f20cea086e8c02cc3e13c8657003.

Note: I removed the `explicit` on the move constructor of Array to make it usable in some use cases.
Is it acceptable?

Reply:
I am going to click the "reslove" button for I didn't receive further feedback.